ocaml/xenopsd/lib/xenops_server.ml

```diff
@@ -2612,13 +2676,24 @@ and perform_exn ?subtask ?result (op : operation) (t : Xenops_task.task_handle)
                       ; [VGPU_start (vgpus, true)]
                       ]
               in
+              let mem_fd =
```

Comment:
Here, `mem_fd` _is_ accurately referring to the fd that the memory is transferred over.